osu.Game/Online/Multiplayer/MultiplayerClient.cs

```diff
@@ -534,7 +535,24 @@ Task IMultiplayerClient.MatchRoomStateChanged(MatchRoomState state)
 
         public Task MatchEvent(MatchServerEvent e)
         {
-            // not used by any match types just yet.
+            if (Room == null)
+                return Task.CompletedTask;
+
+            Scheduler.Add(() =>
+            {
+                if (Room == null)
+                    return;
+
+                switch (e)
+                {
+                    case CountdownChangedEvent countdownChangedEvent:
+                        Room.Countdown = countdownChangedEvent.Countdown;
+                        break;
+                }
+
+                RoomUpdated?.Invoke();
```

Comment:
I find it a touch weird that this comes in as an event but is then diluted down to a `RoomUpdated` call. But I guess this is in line with what we are doing in other places for simplicity of client-side UI updates?

Reply:
Anything updating the room state goes via `RoomUpdated`. Which is pretty much all callbacks.

I originally also exposed an `NewEvent` event, but it wouldn't be much use in this case because we also need to respond to the initial room state, so we'd have to use a common method between countdown and room updates in the ready button anyway.